apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerConnection.cs

```diff
@@ -289,16 +289,13 @@ public override void Close()
         /// Creates a new <see cref="SpannerCommand"/> to delete rows from a Spanner database table.
         /// </summary>
         /// <param name="databaseTable">The name of the table from which to delete rows. Must not be null.</param>
-        /// <param name="deleteFilterParameters">Filter criteria to control what rows get deleted.
-        /// The name of each <see cref="SpannerParameter"/> should match the column name from the Spanner Table.
-        /// The value of the parameter should be the value to filter for the delete operation.
-        /// May be null.</param>
+        /// <param name="primaryKeys">The set of columns that form the primary key of the table.</param>
         /// <returns>A configured <see cref="SpannerCommand"/></returns>
         public SpannerCommand CreateDeleteCommand(
             string databaseTable,
-            SpannerParameterCollection deleteFilterParameters = null) => new SpannerCommand(
+            SpannerParameterCollection primaryKeys = null) => new SpannerCommand(
```

Comment:
This is just a name change to the argument that indicates you cannot simply pass *any* set of filter criteria.  You must pass the set of columns (name + values) that are part of the primary key.
You can still think of it as a deletefilter -- but its more restrictive.